src/app/app.component.html

```diff
@@ -15,11 +15,15 @@
 
 <mat-spinner *ngIf="loading; else loadedConfig"></mat-spinner>
 <ng-template #loadedConfig>
-  <mat-toolbar color="primary">
-    <mat-icon>library_books</mat-icon>
+  <mat-toolbar color="primary" id="pa_header">
+    <a href="https://platform.planqk.de/my-content">
+    <img src="https://platform.planqk.de/img/PlanQK_Logo_Weiss_Beta_Version.5b689144.svg" alt="PlanQK Logo" width="250"
+         style="max-width: 7rem; margin-top: 1rem">
+    </a>
     <span style="margin-left: 1rem">Pattern Atlas</span>
     <span class="spacer"></span>
     {{welcomeText}}
+    <a mat-stroked-button href="https://platform.planqk.de/my-content">Back to the PlanQK platform</a>
```

Comment:
I guess we should make this configurable. In the local or other environment, we should not show this. Thus, maybe add a corresponding flag which only shows this if the the flag is set.